ts/session/utils/sync/syncUtils.ts

```diff
@@ -83,8 +80,8 @@ const buildSyncVisibleMessage = (
     quote,
     preview,
     syncTarget,
-    expireTimer: expireUpdate?.expirationTimer || dataMessageExpireTimer,
-    expirationType: expireUpdate?.expirationType || null,
+    expireTimer: expireUpdate.expirationTimer || 0,
+    expirationType: expireUpdate.expirationType || 'unknown',
```

Comment:
Since the `expireUpdate` is no longer optional I think you can do this:

```tsx
expireTimer: expireUpdate.expirationTimer,
expirationType: expireUpdate.expirationType,
```

Reply:
fixed (multiple times)